tests/components/zwave_js/test_sensor.py

```diff
@@ -779,6 +781,34 @@ async def test_energy_production_sensors(
         assert state.state == str(state_data["state"])
         for attr, val in state_data["attributes"].items():
             assert state.attributes[attr] == val
-
         for attr in state_data.get("missing_attributes", []):
             assert attr not in state.attributes
+
+    entity_id = "sensor.node_2_energy_production_today"
+    state = hass.states.get(entity_id)
+    assert state
+    assert "last_reset" not in state.attributes
+    event = Event(
+        "value updated",
+        {
+            "source": "node",
+            "event": "value updated",
+            "nodeId": energy_production.node_id,
+            "args": {
+                "commandClassName": "Energy Production",
+                "commandClass": 144,
+                "endpoint": 0,
+                "property": "value",
+                "propertyName": "value",
+                "propertyKey": 2,
+                "propertyKeyName": "2",
+                "newValue": 1.0,
+                "prevValue": 123.45,
+            },
+        },
+    )
+    with patch("homeassistant.util.dt.now", return_value=dt_util.now()):
```

Comment:
Nowadays we use the freeze gun helper instead I think.

Reply:
good to know but this will be removed per comment above